### ISSUE TITLE
Quest ID 1638 Electrolyzing Water (ProjectRed) Description Change

### ISSUE THREAD

**Issue by shawnbyday** [GTNewHorizons/NewHorizons#4180]
#### Which modpack version are you using?
2.0.6.7c
#
#### If in multiplayer; On which server does this happen? 
Private
#
#### What did you try to do, and what did you expect to happen?
Followed description to setup Hydrogen and Oxygen automation with ProjectRed.

![image](https://user-images.githubusercontent.com/33701188/53677642-f5420f00-3c78-11e9-8035-1289c726bbae.png)

Quest has you put Stock Keeper Chip and Item Extractor Chip on the Electrolyzer. Then on the Oxygen Super Tank it has you put Item Responder Chip and Item Extractor Chip on it.

I expected the loop to work.
#
#### What happend instead? (Attach screenshots if needed)
Electrolyzer would craft the Oxygen and Hydrogen and push the Hydrogen to the Super Tank via GT Fluid Pipes and ProjectRed would transfer the Oxygen Cells to the Super Tank and empty them. After that the Empty Cells would just sit in the Super Tank and not return to the Electrolyzer.
#
#### What do you suggest instead/what changes do you propose?
The quest description needs to be changed  so that it tells you to do the following:
- Electrolyzer = Stock Keeper Chip and Item Extractor Chip
- Super Tank (Oxygen) = Item Broadcaster Chip and Item Responder Chip

The quest Retrieval Task should be changed to:
- 1x Basic Electrolyzer
- 1x Item Stock Keeper Chip
- 1x Item Broadcaster Chip
- 1x Item Extractor Chip
- 1x Item Responder Chip
- 2x Routed Interface Pipe (Instead of 3 since you don't need a chest for this loop to work)

Basically for the Super Tank you need to swap the Item Extractor Chip out for the Item Broadcaster Chip.

**Comment by dragagon** [comment]
This defeats the purpose of the other quests where you put an empty black-list item responder onto your drawer controller so the cans go back into the system rather than have multiple broadcasters.

**Comment by Dream-Master** [comment]
@shawnbyday so did this need a change or not?

**Comment by richardhendricks** [comment]
I think the quest is ok as is.  

The problem is that there is a difference between broadcasting you *accept* something and a stock keeper telling the network it *wants* something. Extractors only respond to something accepting the item, vs a stock keeper that only responds to broadcasters telling the network something is available.
